verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -617,16 +627,21 @@ void IndexingFactsTreeExtractor::ExtractClassInstances(
   //
   // Loop through each instance and associate each declared id with the same
   // type and create its corresponding facts tree node.
-  for (const TreeSearchMatch& instance : register_variables) {
-    IndexingNodeData indexing_node_data(IndexingFactType::kClassInstance);
-
-    const verible::TokenInfo& variable_name =
-        GetInstanceNameTokenInfoFromRegisterVariable(*instance.match);
-
-    const Anchor variable_name_anchor(variable_name, context_.base);
-    indexing_node_data.AppendAnchor(variable_name_anchor);
+  for (const TreeSearchMatch& instance : class_instances) {
+    const auto tag = static_cast<verilog::NodeEnum>(instance.match->Tag().tag);
+    if (tag == NodeEnum::kRegisterVariable) {
+      const verible::TokenInfo& instance_name =
+          GetInstanceNameTokenInfoFromRegisterVariable(*instance.match);
 
-    class_node.NewChild(indexing_node_data);
+      class_node.NewChild(
+          IndexingNodeData({Anchor(instance_name, context_.base)},
+                           IndexingFactType::kClassInstance));
+    } else if (tag == NodeEnum::kVariableDeclarationAssignment) {
+      const SyntaxTreeLeaf& leaf =
+          GetUnqualifiedIdFromVariableDeclarationAssignment(*instance.match);
+      class_node.NewChild(IndexingNodeData({Anchor(leaf.get(), context_.base)},
+                                           IndexingFactType::kClassInstance));
```

Comment:
Our coverage reports show these lines weren't covered. Is it easy to add a test case that exercises these?